doc/source/deploy/environment-index.rst

```diff
@@ -1,3 +1,5 @@
+.. _env-index:
```

Comment:
This is a generated file, these are going to be overwritten the next time the env-generator gets run.

However, I like the idea of adding anchors to all of these entries. I will take a todo to add that to the tool.